api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -567,28 +558,6 @@ private void verifyCaptcha(String captchaToken) {
     }
   }
 
-  private void verifyInstitutionalAffiliation(Profile profile) {
```

Comment:
pushed this down into the Service layer. Ideally more things would be there but that would take a LOT of time for a relatively time sensitive PR.